platform/security/sts/security-sts-propertyclaimshandler/src/test/resources/users.properties

```diff
@@ -31,5 +31,7 @@
 #
 
 # DDF Changes: changed username/pw to admin/admin, added localhost certificate user
-admin=admin,group,admin,manager,viewer
-localhost=localhost,group,admin,manager,viewer,codice-history,localhost-data-manager
\ No newline at end of file
+admin=admin,group,admin,manager,viewer,_g_:mygroup
+localhost=localhost,group,admin,manager,viewer,codice-history,localhost-data-manager
+adam=changeit,group,_g_:mygroup
```

Comment:
❓ Do we need these last two entries? If so, change `adam` to something more generic.

Reply:
We certainly need the \_g\_: entry to test the original issue. I put the `adam `example in to test the case where a user's roles was completely defined by a group so I think that ought to stay too. However, I'll change the name to user1

Reply:
Alternatively it could be a part of the hero testing instructions to add those entries. If two committers are good with them being in there by default that works too